composer.json

```diff
@@ -83,8 +83,7 @@
 	},
 	"scripts":{
 		"test": [
-			"@analyze",
-			"@phpunit"
```

Comment:
This also changes the behavior for people that run `composer test` locally. I suspect the issue you are fixing comes from the GH Actions config

Reply:
Yup. Unless https://github.com/gesinn-it-pub/docker-compose-ci/commit/08783a070080fa0bb519508dd7b1c81fd9c78f88 gets reverted.

Reply:
The readme specifies > This extension provides unit and integration tests and is usually run by a [continues integration platform](https://docs.github.com/en/actions) but can also be executed locally using the shortcut command composer phpunit from the extension base directory.

Anyways. This unblocks the CI, so either this should be merged or https://github.com/gesinn-it-pub/docker-compose-ci/commit/08783a070080fa0bb519508dd7b1c81fd9c78f88 reverted.

Reply:
I think we should revert https://github.com/gesinn-it-pub/docker-compose-ci/commit/08783a070080fa0bb519508dd7b1c81fd9c78f88

Reply:
This is mainly because many other extensions are using docker-compose-ci. Forcing them to change their compose setup is not ideal.